packages/memprocfs.vm/tools/chocolateyinstall.ps1

```diff
@@ -3,8 +3,41 @@ Import-Module vm.common -Force -DisableNameChecking
 
 $toolName = 'MemProcFS'
 $category = VM-Get-Category($MyInvocation.MyCommand.Definition)
+# Get latest version from GitHub releases with a curl equivalent in powershell
+$latestFileName = 'MemProcFS_files_and_binaries-win_x64-latest.zip'
+$latestVersionUrl = 'https://api.github.com/repos/ufrisk/MemProcFS/releases/latest'
```

Comment:
I have concerns about using `https://api.github.com/repos/ufrisk/MemProcFS/releases/latest`, as this means we potentially update to major versions without any tool testing, which could cause issues with newer dependency requirements. I think we should use a concrete version and allow our bot to update it so that we can test it by using `https://github.com/ufrisk/MemProcFS/releases/download/v5.15/MemProcFS_files_and_binaries-win_x64-latest.zip`.